.github/workflows/lint_test_build.yml

```diff
@@ -0,0 +1,34 @@
+name: Lint+Test+Build
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
I'm not sure what main is. It's probably the same as master right ?

Since you're specifying the same action on push and pull request you can just use the push one.

If you want to run on every push for example you can use this :
```
on:
  [push]
```

I

Reply:
yeah main and master is basically same, github defaulted it to "main" lately. 

I specifically wanted to make sure that forked repository pull request also works. So explicitly specified "pull request" there.